repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -736,9 +749,14 @@ export interface CapDetails {
 }
 
 export interface CapInfo {
```

Comment:
I dont think we should have the ID of the cap inside the actual cap, because if we look at the PremadeCap interface, the id is there. We dont need the id to be inside the cap as well as outside.

Reply:
no longer using pre made cap unless it is in the db function